packages/clerk-js/src/core/fraudProtection.ts

```diff
@@ -20,6 +24,13 @@ export class FraudProtection {
   ) {}
 
   public async execute<T extends () => Promise<any>, R = Awaited<ReturnType<T>>>(clerk: Clerk, cb: T): Promise<R> {
+    if (this.hasReachedDeadEnd) {
+      throw new ClerkRuntimeError(
+        'Security verification failed. Please try again by refreshing the page, clearing your browser cookies, or using a different web browser.',
+        { code: 'captcha_client_dead_end' },
```

Comment:
"dead end" seems a bit of a wild term.
```suggestion
        { code: 'captcha_client_attempts_exceeded' },
```